src/Infrastructure/Persistence/Doctrine/Repository/Regulation/RegulationOrderRepository.php

```diff
@@ -54,4 +54,17 @@ public function getDuplicateIdentifier(string $identifier, Organization $organiz
 
         return \sprintf('%s-%d', $identifier, $nextNumber);
     }
+
+    public function findVisasAndReasonsByRegulationOrderUuid(string $uuid): array
```

Comment:
Vu `getOneOrNull`, ça peut ernvoyer `null` aussi ? (si le uuid n'existe pas)

```suggestion
    public function findVisasAndReasonsByRegulationOrderUuid(string $uuid): ?array
```

Reply:
Nop, à cause du `select` ca retourne un tableau vide